album.go

```diff
@@ -24,16 +26,18 @@ func (client *Client) GetAlbum(ctx context.Context, params ...mxmParams.Param) (
 
 }
 
-/*Get the list of the songs of an album.
+/*
+Get the list of the songs of an album.
 
 Parameters:
-    AlbumID   - Musixmatch album id
-    AlbumMbId - Musicbrainz album id
-    HasLyrics - When set, filter only contents with lyrics
-    Page      - Define the page number for paginated results
-    PageSize  - Define the page size for paginated results. Range is 1 to 100.
+
+	AlbumID   - Musixmatch album id
+	AlbumMbId - Musicbrainz album id
+	HasLyrics - When set, filter only contents with lyrics
+	Page      - Define the page number for paginated results
+	PageSize  - Define the page size for paginated results. Range is 1 to 100.
 */
-func (client *Client) GetAlbumTracks(ctx context.Context, params ...mxmParams.Param) (*[]Track, error) {
+func (client *Client) GetAlbumTracks(ctx context.Context, params ...mxmParams.Param) ([]*Track, error) {
```

Comment:
In the other functions, we return a pointer of Task. So, it makes sense to return a slice to Track pointer